### PR TITLE
Change logo text color

### PR DESCRIPTION
**Pull request by Seung-hwan285**
## Description
The #333 color in the design is not very visible from the user's point of view. I was looking for a green color similar to the logo color, and then I found a color that matches the dark mode and applied it.

I wrote CHANGELOG.
However, there was no record of ## [unpublished] in my log below.
<br>
change

## Fixes (issue)
Fixed logo text to be more visible in dark mode.

## Type of change
- [x] Other (Change logo text color)

Test<br>
white
<br>
dark
<br>
MobileDark

## Checklist:
- [x] I have created my branch from a recent version of `master`
- [x] The code is clean and easy to understand
- [x] I have written tests to guarantee that everything is working properly
- [x] I have made corresponding changes to the documentation
- [x] I have added changes to the `Unreleased` section of the CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.4.1] - 2022-03-12
+
+### Changed
+
+- Changed logo text color from #333 to #44533f
```
Comment (barfurth) [inline CHANGELOG.md:14]: This change does not need to be in the Changelog, please reset this file to its original state.

File: docs/assets/logo.svg [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
-            fill="#333" />
+
+              fill="#44533f" /> <!--Previous fill="#333"-->
```
Comment (barfurth) [inline docs/assets/logo.svg:5]: The comment here is unnecessary, we can see the old color in the Git history.

**Review by barfurth — CHANGES_REQUESTED** [review]
Hey, thanks for this PR. I personally like the color. I will leave the decision to @imbrn of course.

In the meantime please reset the changelog file as this change does not need to be in there. We will
mention you in the release notes if this change is accepted.

Please remove the `.idea` folder.

File: docs/assets/logo.svg (hunk 2 above)
Comment (Seung-hwan285) [reply docs/assets/logo.svg:5]: thank you! It's my first time contributing to an open source, so it's lacking a lot, but thank you for your thorough review!
I've edited your review now.

Comment (Seung-hwan285) [reply docs/assets/logo.svg:5]: @barfurth  I'm sorry. Is it ok to cancel and send the request again? It seems that the current files are kinked.

**Comment by barfurth** [comment]
@Seung-hwan285 closing this as per your request. Feel free to resubmit cleanly.

**Pull request closed without merging**